x/leverage/keeper/supply.go

```diff
@@ -2,20 +2,19 @@ package keeper
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 
 	"github.com/umee-network/umee/v2/x/leverage/types"
 )
 
 // GetSupplied returns an sdk.Coin representing how much of a given denom a
 // user has supplied, including interest accrued.
 func (k Keeper) GetSupplied(ctx sdk.Context, supplierAddr sdk.AccAddress, denom string) (sdk.Coin, error) {
-	if !k.IsAcceptedToken(ctx, denom) {
-		return sdk.Coin{}, sdkerrors.Wrap(types.ErrInvalidAsset, denom)
+	if types.HasUTokenPrefix(denom) {
+		return sdk.Coin{}, types.ErrUToken.Wrap(denom)
```

Comment:
Same as mentioned earlier - this allows blacklisted assets to still be queried. (Transactions unaffected.)